Library/Formula/devil.rb

```diff
@@ -1,11 +1,53 @@
 class Devil < Formula
   desc "Cross-platform image library"
   homepage "https://sourceforge.net/projects/openil/"
-  url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/DevIL-1.7.8.tar.gz"
-  sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
   revision 1
 
-  bottle :disable, "Can't generate bottles until builds with either Clang or GCC-5"
+  stable do
+    url "https://downloads.sourceforge.net/project/openil/DevIL/1.7.8/" \
+      "DevIL-1.7.8.tar.gz"
+
+    sha256 "682ffa3fc894686156337b8ce473c954bf3f4fb0f3ecac159c73db632d28a8fd"
+
+    # fix compilation issue for iluc.c
+    patch \
+      "--- a/src-ILU/ilur/ilur.c 2009-03-08 08:10:12.000000000 +0100\n" \
+      "+++ b/src-ILU/ilur/ilur.c 2010-09-26 20:01:45.000000000 +0200\n" \
+      "@@ -1,6 +1,7 @@\n" \
+      " #include <string.h>\n" \
+      " #include <stdio.h>\n" \
+      "-#include <malloc.h>\n" \
+      "+#include <stdlib.h>\n" \
+      "+#include \"sys/malloc.h\"\n" \
+      "\n" \
+      " #include <IL/il.h>\n" \
+      " #include <IL/ilu.h>\n"
+  end
+
+  head do
+    url "https://github.com/DentonW/DevIL.git"
+
+    depends_on "automake" => :run
```

Comment:
Oh, yep, that makes sense. That's probably a flaw in our dependency resolution, because automake "runs" at build time for other formulae that use it. But when you're building a different formula, that `=> :run` dependency is bypassed.

How about switching this to `automake => :build` and adding a `depends_on 'autoconf' => :build`. Does that work?